### PR TITLE
A few more options

### PR DESCRIPTION
**Pull request by etlweather**
Added options for lease-time by pool as well as the overall authoritative declaration for the server.

### CODE REVIEW

File: manifests/pool.pp [hunk 1]
```diff
@@ -9,6 +9,8 @@
   $pxeserver   = undef,
   $domain_name = undef,
   $static_routes = undef,
+  $default_lease_time = undef,
```
Comment (ekohl) [inline manifests/pool.pp]: Can we validate these as integers or does dhcpd allow these to be non-integers? If so:
```puppet
if $default_lease_time {
    validate_integer($default_lease_time)
}
```

**Comment by ekohl** [comment]
In general :+1:, just some minor touches. We also prefer a single squashed commit unless it makes logical sense to split it.

**Comment by ekohl** [comment]
I took the two authoritative commits, squashed them and added a simple test. I'm not sure about the other commit since you can already add them via ```parameters``` (as an array), but it doesn't hurt I suppose.

**Comment by mmoll** [comment]
@etlweather ping?

**Comment by etlweather** [comment]
@mmoll  ?

**Comment by mmoll** [comment]
@etlweather as @ekohl said, a part of this PR already went in, for the other a rebase onto current master is needed anyway.

**Comment by mmoll** [comment]
Also, please squash your commits together into one.

**Comment by mmoll** [comment]
no reaction -> closing.

**Pull request closed without merging**